packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -81,6 +81,7 @@ export async function bundleServer(buildOpts: BuildOptions): Promise<void> {
 		minifySyntax: !debug,
 		legalComments: "none",
 		metafile: true,
+		sourcemap: getOpenNextConfig(buildOpts).cloudflare?.sourcemap,
```

Comment:
May be we can always generate the source maps.

The Next build dominates the build time so the build time should not change a lot

```suggestion
		sourcemap: true,
```